util/dvsim/FlowCfg.py

```diff
@@ -36,6 +36,7 @@ def __init__(self, flow_cfg_file, proj_root, args):
         self.scratch_root = args.scratch_root
         self.branch = args.branch
         self.job_prefix = args.job_prefix
+        self.duts = args.duts
```

Comment:
Change this to `select_cfgs`.
Also, create a local copy in FlowCfg instance rather than pass the list handle from `args`.   
`select_cfgs` should be treated very similar to `items`.